ROADMAP.md

```diff
@@ -0,0 +1,23 @@
+# Roadmap
+
+## Top-level goals
+
+* Extensibility: custom components, shaders
+* Performance: raycasting (three.js), head tracking on mobile (webvr-polyfill), anything DOM/Custom Elements-related
+* Ecosystem: enable discovery and distribution of assets, components, scenes, snippets, shaders
+
+## `0.2.0`
```

Comment:
This seems pretty redundant with the CHANGELOG, I think the ROADMAP should just be talking about very forward-facing.